scripts/dev/templates/Dockerfile.linting

```diff
@@ -0,0 +1,7 @@
+FROM {{base_image}}
+
+ADD . .
```

Comment:
do we need to add everything? We can reduce the context size by just including the python files

Reply:
> I think black is only a Python formatter, but not linter. For linting we should use pylint.

This is a good point. How would a linter fit in the process in our case? 
As it just outputs diagnostics we can't automate the process as with a formatter.

Reply:
Yes we could do that. At the moment we are actually testing three directories (`test`,`pkg` and `scripts`) but should we assume that python scripts will always be in the latter? in this case we could just add that.

Reply:
I decided to go with listing all directories that contain python scripts and add them one by one

Reply:
(actually, I modified it to add all the files one by one, not the directories)